docs/docker/intro.md

```diff
@@ -0,0 +1,33 @@
+# Docker
+
+Docker is a container based virtualization software that allow it's user to create a development environment in a easy way without installing a virtualization software i.e VirtualBox.
+
+This allow developer to create multiple test environment on the fly and to share with others without too much headaches.
+
+# Architecture
+
+Docker architecture is centralized on the daemon. The daemon is used by all of the component of Docker. 
+
+In order to control all of these components Docker is using a rest api endpoint that is used by the docker cli. This CLI will send request to the api endpoint which will then be analyze by the docker daemon.
```

Comment:
In order to control all of those components, Docker uses a rest api endpoint that is also used by the docker CLI. This CLI will send requests to the api endpoint which will then be processed by the docker daemon.

Reply:
Done